packages/react/src/components/Blockquote/Blockquote.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { BlockquoteProps } from "./Blockquote.props";
+import useGlobalSettings from "../../hooks/useGlobalSettings";
+
+const Blockquote: React.FC<BlockquoteProps> = ({ quote, cite }) => {
+  const { prefix } = useGlobalSettings();
+
+  return (
+    <blockquote className={`${prefix}--blockquote`}>
+      <p>{quote}</p>
+      <cite>{cite}</cite>
+    </blockquote>
+  );
+};
+
+export default Blockquote;
```

Comment:
And let's stick to the named exports, the default exports can't be type augmented 

https://github.com/Microsoft/TypeScript/issues/14080